src/app/clients/client-stepper/client-general-step/client-general-step.component.ts

```diff
@@ -117,7 +117,7 @@ export class ClientGeneralStepComponent implements OnInit {
         this.createClientForm.removeControl('lastname');
         this.createClientForm.addControl('fullname', new FormControl('', [Validators.required, Validators.pattern('(^[A-z]).*')]));
         this.createClientForm.addControl('clientNonPersonDetails', this.formBuilder.group({
-          'constitutionId': [''],
+          'constitutionId': ['', Validators.required],
```

Comment:
Why is it required? It is not required at backend.

Reply:
It's required in the backend in the case of Charge for Client entities

Reply:
Yeah...you are right. I have found the validation....it was just at a weird place...